module/zfs/vdev_indirect.c

```diff
@@ -1632,7 +1632,7 @@ vdev_indirect_splits_damage(indirect_vsd_t *iv, zio_t *zio)
 		}
 
 		iv->iv_attempts_max *= 2;
-		if (iv->iv_attempts_max > (1ULL << 16)) {
+		if (iv->iv_attempts_max >= (1ULL << 12)) {
```

Comment:
is this related to the ztest fix? Or is this just making it run faster by not damaging when it would cause more than 2^12 tries?

Reply:
See comment above. with the existing code we can have 17 damaged segments which was causing the ztest deadman to trigger fairly often.